README.md

```diff
@@ -54,39 +54,51 @@ brain atlases obtained with ISH gene expression and Nissl stains.
 
 
 ### Official documentation
-All details related to installation and logic are described in the official documentation. Once
-deployed a link will be added. Until then one can generate it locally following the instructions
-below.
+All details related to installation and logic are described in the 
+[official documentation](https://atlas-alignment.readthedocs.io/).
 
 
-### Normal installation
-Make sure the `pip` version is at least `19.1`.
+### Installation
 
-```bash
+#### Installation Requirements
+
+##### Pip
+First of all, make sure the `pip` version is at least `19.1`.
+```shell script
 pip install --upgrade pip
 pip --version  # make sure >= 19.1
 ```
 
```

Comment:
I think this `pip` update is not necessary anymore because we are dropping the `@` syntax for direct links

Reply:
Sure, having a recent `pip` is always a good thing. However, IMO it is not necessary to install the package.

Reply:
Don't we use it in `tox.ini` at line `13`?
```
    lpips_tf @ git+http://github.com/alexlee-gk/lpips-tensorflow.git#egg=lpips_tf
```

Reply:
Hmmm. The thing is that `tox` will have it's own `pip` and it is not going to be the one that we instruct the user to update in the README (the one that lies in our virtual environment).

See https://github.com/tox-dev/tox/issues/791 for reference.

Reply:
Fixed this + docs in 10aa518